.travis.yml

```diff
@@ -2,7 +2,7 @@ language: python
 python:
   - "2.6"
   - "2.7"
-script: make tests
+script: make tests-travis
```

Comment:
Which services need to run for the integration tests? We should also aim to hook it up on travis asap.